src/state/user/reducer.ts

```diff
@@ -135,6 +135,11 @@ const userSlice = createSlice({
       if (state.selectedWallet === 'UNIWALLET') {
         state.selectedWallet = ConnectionType.UNISWAP_WALLET
       }
+      // WALLET_CONNECT (v1) has been deprecated
+      // @ts-expect-error
+      if (state.selectedWallet === 'WALLET_CONNECT') {
+        state.selectedWallet = ConnectionType.UNISWAP_WALLET
```

Comment:
Maybe you meant to set to V2 here, but I think we'd want to just set to undefined

```suggestion
        state.selectedWallet = undefined
```

Reply:
good catch!